.github/workflows/aws-lambda.yml

```diff
@@ -0,0 +1,38 @@
+name: AWS Lambda tests
+
+permissions: read-all
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install libcurl4-openssl-dev and net-tools
+      run: |
+        sudo apt-get update
+        sudo apt-get install libcurl4-openssl-dev
+        sudo apt-get install net-tools
+
+    - run: sam --version
+
+    - uses: actions/checkout@v2
```

Comment:
should we be pinning these to a specific sha?

Reply:
Good idea, done 👍